lib/pymedphys/docs/release-notes.md

```diff
@@ -8,6 +8,87 @@ This project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [0.35.0]
+
+### News around this release
+
+* Installation on MacOS (Intel) has been simplified and is now the same as for other platforms, thanks to work by the new pymssql maintainers.
+
+### "Stable" API changes
+
+(Won't truly be stable until after a 1.0.0 release)
+
+#### Breaking changes
+
+* DicomListener removed from public API (access is only through CLI)
+
+#### Deprecations
+
+
+#### New features
+
+* `pymedphys dicom listen` stores incoming DICOM objects in a directory hierarchy: PatientID/Study Instance UID/Series Instance UID
+aligning with the DICOM Q/R hierarchy, and more suitable for use with tools like dicompyler and OnkoDICOM
```

Comment:
If this was in regular use, this possibly is a breaking change. I'm not sure if we should consider "how we store data" part of our API... What do you think?

Reply:
It isn't a change in the API (Application Programmer Interface) itself, it isn't a change in the CLI either.

We can't say for sure it would be a breaking change.
If someone is looking at output/* then yes.  If they are globbing output/** (maybe output/**/*?) then it isn't.
But it is worth warning the user.

I'm not sure how regular the use could be given that 0.34 was released quite recently.

Reply:
I guess I was sort of thinking of the file output from the CLI being the "output"/"return" API of the CLI. I think if we go down that path we're going to trigger breaking API changes for all sorts of things. I'm happy to not have it be declared as part of our API.